website/src/documentation/programming-guide.md

```diff
@@ -3121,6 +3121,15 @@ public class MyMetricsDoFn extends DoFn<Integer, Integer> {
   }
 }
 ```  
+### 9.5 Export metrics {#export-metrics}
+Beam metrics can be exported to external sinks. If set up the runner will push metrics to a configured sink at a configured period. 
+The configuration is held in the [MetricsOptions](https://beam.apache.org/releases/javadoc/2.19.0/org/apache/beam/sdk/metrics/MetricsOptions.html) class.
+It contains push period configuration and also sink specific options such as URL. As for now only REST http sink and graphite sinks are supported and only
+Flink and Spark runners support metrics export.
+
+Also Beam metrics are exported to inner Spark and Flink dashboards to be consulted in their respective UI.
+
+
```

Comment:
Do you have any code examples of metrics export? It will worth to add if any.

Reply:
there is no more code than what is present in the example in previous sections. The user only needs to configure the export to export the metrics he defined in his pipeline, what do you mean ?

Reply:
I was thinking about any useful examples that will help users to use it.